### PR TITLE
docs: Migration Guide, F

### PR DESCRIPTION
**Pull request by eranhirsch**
There's a lot of functions starting with F which had changes

### CODE REVIEW

File: docs/src/content/v1-migration/find.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Typing
+
+When using with a type-predicate, the returned item would now be narrowed to the
+type of the type-predicate, similar to how `filter` with a type-predicate
+narrows the resulting items in the array.
+
```
Comment (cjquines) [inline docs/src/content/v1-migration/find.md:6]: **nit:** let's make this second paragraph to match example order, and for consistency with other `indexed` changes...

File: docs/src/content/v1-migration/forEachObj.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  console.log(`${key} => ${value}`);
+});
+
+// Now
+forEachObj({ a: 1, b: 2 }, (value, key) => {
+  console.log(`${key} => ${value}`);
+});
+```
+
+### Symbol Keys
```
Comment (cjquines) [inline docs/src/content/v1-migration/forEachObj.md:40]: **nit:** `Keys` -> `keys`

Comment (cjquines) [reply docs/src/content/v1-migration/forEachObj.md:40]: (our headings are in sentence case)

Comment (cjquines) [reply docs/src/content/v1-migration/forEachObj.md:40]: lmao this is an extremely tiny nit, i will probably just give it a copyediting pass when they're all written

Comment (eranhirsch) [reply docs/src/content/v1-migration/forEachObj.md]: These become repetitive to write and are quite draining for me, I'm happy that another person is reading them and getting a feel for them so they stay consistent.

**Pull request merged**